composer.json

```diff
@@ -15,7 +15,7 @@
         }
     ],
     "require": {
-        "php": "^7.1.3 || ^8.0",
+        "php": ">=7.2",
```

Comment:
Please revert this change.

Reply:
`PHP_OS_FAMILY` constant is available from PHP 7.2 up. I can change the code to use `PHP_OS`. I'll work on that tonight.

Reply:
Pull in the symfony php 7.2 polyfill, please.

Reply:
I used `PHP_OS` since we have the same result without an extra dependency.